Makefile

```diff
@@ -678,6 +678,10 @@ test-e2e: ## Run "docker-build" and "docker-push" rules then run e2e tests.
 	$(MAKE) docker-build docker-push \
 	test-e2e-run
 
+create-bootstrap-cluster:
```

Comment:
how is this different from the `kind-create` target?

Reply:
Almost similar -- just a bit of difference is that `kind-create` is used by tilt and in this target we explicitly set the cluster name to be capz-e2e. 
I kept those different to kind of segregate the local dev and e2e scenarios.